Solutions/SpyCloud Enterprise Protection/Analytics Rules/SpyCloudEnterpriseProtectionBreachRule.yaml

```diff
@@ -0,0 +1,53 @@
+id: cb410ad5-6e9d-4278-b963-1e3af205d680
+name: SpyCloud Enterprise - Malware Detection
+description: |
+  'This alert creates an incident when an malware record is detected in the SpyCloud watchlist data'
+severity: High
+status: Available
+requiredDataConnectors: []
+queryFrequency: 12h
+queryPeriod: 12h
+triggerOperator: gt
+triggerThreshold: 0
+suppressionDuration: 5h
+tactics: []
+relevantTechniques: []
+query: |
+  SpyCloudTest_CL
+  | where Severity_s == '25'
+  | project TimeGenerated, Document_Id_g, Source_Id_s, SpyCloud_Publish_Date_t, Email_s, Domain_s, Password_s, Password_Plaintext_s, Username_s, IP_Address_s
+incidentConfiguration:
+  createIncident: true
+  groupingConfiguration:
+    enabled: true
+    reopenClosedIncident: false
+    lookbackDuration: 12h
+    matchingMethod: AllEntities
+    groupByEntities: []
+    groupByAlertDetails:[]
+    groupByCustomDetails: []

```

Comment:
Please remove "groupByCustomDetails: []" in both analytic rules, if you are not using it.

Reply:
Removed